src/Main.js

```diff
@@ -0,0 +1,15 @@
+import { Component } from 'react';
+import HornedBeast from './HornedBeast'
+
+class Main extends Component {
+  render() {
+    return (
+  <div>
+    <HornedBeast title="Water Buffalo" img_url="https://upload.wikimedia.org/wikipedia/commons/c/cf/Serengeti_Bueffel2.jpg" description="Mustache head"/>
+    <HornedBeast title="Narwhal" img_url="https://previews.123rf.com/images/lineartestpilot/lineartestpilot1909/lineartestpilot190907578/129368587-rainbow-gradient-line-drawing-of-a-cartoon-narwhal.jpg" description="spooky narwhal"/>
+  </div>
```

Comment:
Wanted to give ya a bit more feedback here regarding indendation:

- after return, depending on your preference for spacing (2 or 4), we can indent the <div> in that many spaces 👍